sycl/test-e2e/ESIMD/local_accessor_block_load_store.cpp

```diff
@@ -9,7 +9,7 @@
 // RUN: %{run} %t.out
 //
 // TODO: Enable the test when GPU driver is ready/fixed.
-// XFAIL: opencl || windows || gpu-intel-pvc
+// XFAIL: opencl || windows || gpu-intel-pvc || gpu-intel-gen12
```

Comment:
This will not work for SYCLOS because gpu-intel-gen12 is not set in SYCLOS CI even for host with gen12.

Reply:
Linux+Gen12 in CI is on the way now. That is why there are patches like this: https://github.com/intel/llvm/pull/10433